benchmark/benchmark.go

```diff
@@ -222,12 +226,25 @@ func NewClientConn(addr string, opts ...grpc.DialOption) *grpc.ClientConn {
 	return conn
 }
 
-func runUnary(b *testing.B, maxConcurrentCalls, reqSize, respSize int) {
+func runUnary(b *testing.B, maxConcurrentCalls, reqSize, respSize, kbps, mtu int, ltc time.Duration) {
 	s := stats.AddStats(b, 38)
+	nw := &latency.Network{Kbps: kbps, Latency: ltc, MTU: mtu}
+	nw.Kbps = -1
 	b.StopTimer()
-	target, stopper := StartServer(ServerInfo{Addr: "localhost:0", Type: "protobuf"}, grpc.MaxConcurrentStreams(uint32(maxConcurrentCalls+1)))
+	target, stopper := StartServer(nw, ServerInfo{Addr: "localhost:0", Type: "protobuf"}, grpc.MaxConcurrentStreams(uint32(maxConcurrentCalls+1)))
 	defer stopper()
-	conn := NewClientConn(target, grpc.WithInsecure())
+	conn := NewClientConn(
+		target,
+		grpc.WithInsecure(),
+		grpc.WithDialer(
+			func(address string, timeout time.Duration) (net.Conn, error) {
+				clientConn, err := nw.TimeoutDialer(net.DialTimeout)("tcp", address, timeout)
```

Comment:
I would prefer to name the returned conn just `conn`. `clientConn` usually means a gRPC client.